app/player/handlers.go

```diff
@@ -0,0 +1,73 @@
+package player
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/lbryio/lbrytv/internal/monitor"
+)
+
+// RequestHandler is a HTTP request handler for player package.
+type RequestHandler struct {
+	player *Player
+}
+
+// NewRequestHandler initializes a HTTP request handler with the provided Player instance.
+func NewRequestHandler(p *Player) *RequestHandler {
+	return &RequestHandler{p}
+}
+
+func (h RequestHandler) getURI(r *http.Request) string {
+	vars := mux.Vars(r)
+	return fmt.Sprintf("%s#%s", vars["uri"], vars["claim"])
+}
+
+func (h RequestHandler) writeErrorResponse(w http.ResponseWriter, s int, msg string) {
+	w.WriteHeader(s)
+	w.Write([]byte(msg))
+}
+
+func (h RequestHandler) processStreamError(w http.ResponseWriter, uri string, err error) {
+	if err == errPaidStream {
```

Comment:
if you're on go 1.13, start using `errors.Is` instead

https://blog.golang.org/go1.13-errors